### PR TITLE
Test fixes for experimental debugger

### PR DESCRIPTION
**Pull request by karthiknadig**

### CODE REVIEW

File: Python/Tests/DebuggerUITests/DebugProjectUITests.cs [hunk 1]
```diff
@@ -123,14 +111,11 @@ public class DebugProjectUITests {
...
             using (new PythonExperimentalGeneralOptionsSetter(pyService, useVsCodeDebugger: useVsCodeDebugger)) {
-                if (useVsCodeDebugger) {
-                    KillPythonProcess(TimeSpan.FromSeconds(30));
-                }
-
                 var sln = app.CopyProjectForTest(@"TestData\RelativeInterpreterPath.sln");
                 var project = app.OpenProject(sln, "Program.py");
                 var interpreterPath = Path.Combine(PathUtils.GetParent(sln), "Interpreter.exe");
-                File.Copy(PythonPaths.Python27.InterpreterPath, interpreterPath, true);
+                var interpreter = PythonPaths.Python27 ?? PythonPaths.Python27_x64;
```
Comment (zooba) [inline Python/Tests/DebuggerUITests/DebugProjectUITests.cs]: Add an `interpreter.AssertInstalled()` call before using it, and put it first thing in the test.

Comment (huguesv) [inline Python/Tests/DebuggerUITests/DebugProjectUITests.cs]: Should assert to get better error message `intepreter.AssertInstalled`

File: Python/Tests/DebuggerUITests/DebugProjectUITests.cs [hunk 2]
```diff
@@ -510,27 +458,63 @@ public class DebugProjectUITests {
...
-                var local = curFrame.Locals.Item("i");
-                Assert.AreEqual("42", local.Value);
+                if (useVsCodeDebugger) {
+                    var locals = new List<Expression>();
+                    foreach (Expression e in curFrame.Locals) {
+                        locals.Add(e);
+                    }
+
+                    var local = locals.Single(e => e.Name == "i");
+                    Assert.IsNotNull(local);
```
Comment (huguesv) [inline Python/Tests/DebuggerUITests/DebugProjectUITests.cs]: `Single` will throw if not found, so no need to check for `null`. Same below.

File: Python/Tests/DebuggerUITests/DebugProjectUITests.cs [hunk 3]
```diff
@@ -539,33 +523,35 @@ public class DebugProjectUITests {
...
-
-                ExceptionTest(app, "SimpleException2.py", "Exception Thrown", "ValueError: bad value", "ValueError", 3);
+                string exceptionDescription = useVsCodeDebugger ? "ValueError('bad value',)" : "ValueError: bad value";
+                ExceptionTest(app, "SimpleException2.py", "Exception Thrown", exceptionDescription, "ValueError", 3);
             }
         }
 
         public void SimpleExceptionUnhandled(VisualStudioApp app, bool useVsCodeDebugger, DotNotWaitOnNormalExit optionSetter) {
+            if (useVsCodeDebugger) {
+                // This should be done as part of jsut my code and the fix for hiding debugger
```
Comment (huguesv) [inline Python/Tests/DebuggerUITests/DebugProjectUITests.cs]: np: typo "just"

**Pull request merged**